### PR TITLE
{vis}[fosscuda/2017b,intelcuda/2017b] matplotlib v2.1.0

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/4bf21eb784ba109d299f1c88764609cf for a full test report.

File: easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-fosscuda-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
+        'checksums': [
+            '4b5f16c9cefde553ea79975305dcaa67c8e13d927b6e55aa14b4a8d867e25387',  # matplotlib-2.1.0.tar.gz
+        ],
+    }),
+]
+
+# specify that PythonBundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
```
Comment (boegel) [inline easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-fosscuda-2017b-Python-2.7.14.eb]: @vanzod Please drop this, it's meaningless now (not picked up by `eb` at all)

File: easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-fosscuda-2017b-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-fosscuda-2017b-Python-2.7.14.eb]: @vanzod This is the default used by `PythonBundle`, so may as well drop it

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/4dca42acf1e6e4797a4b212feda9add7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a910e013de8285f04e31a70159fead79 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2012.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1e8b1e3544613be7e8c705f4f5f0b019 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**